physicalplan/aggregate/scalar_table.go

```diff
@@ -282,7 +283,51 @@ func makeAccumulatorFunc(expr parser.ItemType) (newAccumulatorFunc, error) {
 				},
 			}
 		}, nil
+	case "quantile":
```

Comment:
I think it's best if we add validation here, so that we cannot even instantiate an accumulator with invalid parameters. This way each function can do its own validation.